source/js/download.js

```diff
@@ -0,0 +1,48 @@
+$(document).ready(function($) {
+    'use strict';
+
+    showdown.setFlavor('github');
+    var converter = new showdown.Converter({
+        headerLevelStart: 3
+    }),
+        php_url;
+
+    // Included for local debugging, assuming running from Middleman, default port 4567
+    if (location.port == 4567) {
+        php_url = 'http://localhost:80/github.php';
+    } else {
+        php_url = '/github.php'
+    }
+
+    $.get(php_url, function(data) {
+        var jdata = JSON.parse(data),
+            linux_oc = $('#oc-platforms .linux-platform'),
+            mac_oc = $('#oc-platforms .mac-platform'),
+            windows_oc = $('#oc-platforms .windows-platform'),
+            linux_server = $('#server-platforms .linux-platform'),
+            asset, index, append_html;
+
+        console.log(jdata['assets']);
+        
+        for (var i = 0; i < jdata['assets'].length; i++) {
+            asset = jdata['assets'][i];
+            if ((index = asset['name'].indexOf('client-tools')) >= 0) {
+                append_html = '<a href="' + asset['browser_download_url'] + '">' + asset['name'].substring(index + 13) + '</a><br>';
```

Comment:
`substring(index+13)` does not seem as something that will last working for ages; missing some comments there at least.